Gem/Code/Source/Components/NetworkAnimationComponent.h

```diff
@@ -71,15 +71,15 @@ namespace MultiplayerSample
         EMotionFX::Integration::AnimGraphComponentRequests* m_animationGraph = nullptr;
 
         // Hardcoded parameters, be nice if this was flexible and configurable from within the editor
-        uint32_t m_velocityParamId = InvalidParamIndex;
-        uint32_t m_aimTargetParamId = InvalidParamIndex;
-        uint32_t m_crouchParamId = InvalidParamIndex;
-        uint32_t m_aimingParamId = InvalidParamIndex;
-        uint32_t m_shootParamId = InvalidParamIndex;
-        uint32_t m_jumpParamId = InvalidParamIndex;
-        uint32_t m_fallParamId = InvalidParamIndex;
-        uint32_t m_landParamId = InvalidParamIndex;
-        uint32_t m_hitParamId = InvalidParamIndex;
-        uint32_t m_deathParamId = InvalidParamIndex;
+        size_t m_velocityParamId = InvalidParamIndex;
+        size_t m_aimTargetParamId = InvalidParamIndex;
+        size_t m_crouchParamId = InvalidParamIndex;
+        size_t m_aimingParamId = InvalidParamIndex;
+        size_t m_shootParamId = InvalidParamIndex;
+        size_t m_jumpParamId = InvalidParamIndex;
+        size_t m_fallParamId = InvalidParamIndex;
+        size_t m_landParamId = InvalidParamIndex;
+        size_t m_hitParamId = InvalidParamIndex;
+        size_t m_deathParamId = InvalidParamIndex;
```

Comment:
One easy to solve updating this ever again is with `decltype(InvalidParamIndex)`